src/warnet/cli/network.py

```diff
@@ -108,7 +108,8 @@ def info(network: str):
     Get info about a warnet named [network]
     """
     result = rpc_call("network_info", {"network": network})
-    print_repr(result)
+    if result:
+        print_repr(result)
```

Comment:
75185fb0eb26238e65ac78b33005daf0a9f16790

I think RPC calls should always return *something*

Reply:
Added handling here: https://github.com/mplsgrant/warnet/blob/a68a229af6e90e4111d1ef6461cc8670cc1cdce8/src/warnet/cli/network.py#L114